lecli/deprecated_commands.py

```diff
@@ -0,0 +1,227 @@
+"""Deprecated commands"""
+# pylint: skip-file
+
```

Comment:
I've excluded this file from pylint as the arg/option definitions are triggering the duplicated lines warning. Click requires the args & option definitions on these commands too.